stellargraph/data/explorer.py

```diff
@@ -107,6 +81,114 @@ def run(self, **kwargs):
         """
         raise NotImplementedError
 
+    def _raise_error(self, msg):
+        full = "({}) " + msg
+        raise ValueError(full.format(type(self).__name__))
```

Comment:
This would be better as:

```suggestion
        raise ValueError("({}) {}".format(type(self).__name__, msg))
```

As is, there will be weird errors if `msg` contains a `{` because the `format` will try to interpret that.

Reply:
Good catch. Pointless constructing a temporary, new string anyway, since format will do that for us.